scripts/README.md

```diff
@@ -5,6 +5,14 @@ This folder contains scripts for automating specific tasks.
 
 ## 1. Fetch APIs
 
-[fetch_apis.sh](./fetch_apis.sh) - [fetch_api.sh](./fetch_api.sh) - This script downloads the OpenAPI spec file.
+[fetch_apis.sh](./fetch_apis.sh) - This script downloads the OpenAPI spec file.
 
 
+## 1. Generate
+
+[generate.sh](./generate.sh) - This script generates the API client SDKs following the OpenAPI specs.
+
```

Comment:
these are what the scripts do but they should be called via make targets. maybe just mention that somewhere